docs/authoring.md

```diff
@@ -0,0 +1,151 @@
+# Authoring Components
+
+## Overview
+C#/WinRT provides support for authoring Windows Runtime components. You can write a library in C#, and use C#/WinRT's source generator to get a winmd that any WinRT compatible language can use. For example, a library written in C# can be used by a C++ program, via C#/WinRT and C++/WinRT, with just a few tweaks to the C++ project.
+
+
+## References
+Here are some resources that demonstrate authoring C#/WinRT components and the details discussed in this document.
+1. https://github.com/microsoft/CsWinRT/tree/master/src/Authoring/AuthoringConsumptionTest
+2. https://github.com/AdamBraden/MyRandom
+
+
+## Authoring the C# Component
+To create a library, select the Class Library (.NET Core) template in Visual Studio. C#/WinRT projects require Windows API version specific .NET frameworks.
+
+Accepted `<TargetFramework>` properties |
+--- |
+`net5.0-windows10.0.19041.0` |
+`net5.0-windows10.0.18362.0` |
+`net5.0-windows10.0.17763.0` |
+
+The library you are authoring should specify the following properties in its project file: 
+```
+  <PropertyGroup>

```

Comment:
I found I also needed to add the following line to my PropertyGroup, otherwise my hosting cpp app can't find WinRT.Runtime.dll:
<CopyLocalLockFileAssemblies Condition="$(CsWinRTComponent)==true">true</CopyLocalLockFileAssemblies>

Reply:
nvm - didn't see it was under the section Copying Dlls target. But could also move it to the project file?